theano/scalar/basic.py

```diff
@@ -3838,30 +3868,56 @@ def __init__(self, inputs, outputs):
         # only 1 new Composite each time at the output.
         for i in inputs:
             assert i not in outputs  # This isn't supported, use identity
-        if len(outputs) > 1 or not any([isinstance(var.owner.op, Composite)
-                                        for var in outputs]):
+        if not any([isinstance(var.owner.op, Composite) for var in outputs]):
```

Comment:
While I'm all for the change to support multiple outputs, just before the release is not the right time.  Let's wait until after.